components/Feature/ReferralsTable/index.js

```diff
@@ -30,7 +30,9 @@ const ReferralsTable = ({ referrals }) => {
         <tbody className="govuk-table__body">
           {referrals.map((referral, index) => {
             const recentStatus = getRecentStatus(referral.statusHistory).status;
-            const recentStatusClass = STATUS_MAPPINGS[recentStatus]?.class;
+            const recentStatusClass = STATUS_MAPPINGS[recentStatus]
+              ? STATUS_MAPPINGS[recentStatus].class
+              : '';
```

Comment:
Any way we could upgrade the ES Lint itself? Changing this change makes outdated ES Lint satisfied, but it kind of makes the code harder to read. :thinking: 

Glancing at ES Lint github issues I found people reporting that the problem of it flagging the `?.` syntax goes away with version `7.9.0` & up: https://github.com/eslint/eslint/issues/13566#issuecomment-693380846.

We don't mind es-lint not being backwards compatible, because nothing is running with it as it's base. I imagine at most what will break is the command to run it _(which should be easy enough to change)_ & maybe it flag a couple more things within the code, which arguably is good as that's why we got it in the first place.

Reply:
Actually, thinking about this:

> _"... maybe it flag a couple more things within the code, which arguably is good as that's why we got it in the first place"_

We're are likely missing many things that it would flag us that were only added with later versions. Given that's the entire point of this SQA tool, I think we should 100% update the es-lint to the latest version.

Reply:
That's now upgraded, very good point!